index.d.ts

```diff
@@ -8,6 +8,13 @@ declare namespace elementReady {
 		@default document
 		*/
 		readonly target?: Element | Document;
+
+		/**
+		The time to wait before stopping the search, in milliseconds.
+
+		@default false
+		*/
+		readonly timeout?: number | false
```

Comment:
Should we really do `false`? How about we default to `Infinity` instead?

Reply:
I had initially thought of `0` (which is also falsy).

_Do you think V8 will actually timeout when time reaches infinity? And when it does, do we even? I don't even._

That said, I don't know if `timeout: Infinity` makes sense, but if it's a common pattern: 🤷‍♂️

Reply:
We just check for `Infinity` manually and do the same as we would with `false`. Passing `Infinity` to setTimeout makes it trigger right away, yay JS :P

Reply:
> the same as we would with `false`

`timeout > 0` already filters out `false` so it doesn't need extra checks